### PR TITLE
Feat/footnote

### PR DESCRIPTION
**Pull request by MarioIpedis**
-footnote component
-test 
-readme

### CODE REVIEW

File: packages/footnote/src/components/footnote/footnote.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <span id={`footnote-label-${this.identifier}`} class="footnote-id">
+          [{this.identifier}]
+        </span>
+        <div class="text">
+          <span class="footnote-text">{this.text}</span>
+          <a href={`#ref${this.identifier}`} aria-label="Back to main text">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="24"
+              height="24"
```
Comment (patrickIpedis) [inline packages/footnote/src/components/footnote/footnote.tsx:30]: add aria-hidden=true to the svg

**Review by patrickIpedis — CHANGES_REQUESTED** [review]

File: tsconfig.base.json [hunk 2]
```diff
@@ -22,6 +22,9 @@
       "@ipedis/dropdown": ["dist/packages/dropdown"],
       "@ipedis/dropdown/*": ["dist/packages/dropdown/dist/components/*"],
       "@ipedis/dropdown/loader": ["dist/packages/dropdown/loader"],
+      "@ipedis/footnote": ["dist/libs/footnote"],
+      "@ipedis/footnote/*": ["dist/libs/footnote/dist/components/*"],
+      "@ipedis/footnote/loader": ["dist/libs/footnote/loader"],
```
Comment (yanis-git) [inline tsconfig.base.json:27]: wrong path

File: packages/footnote/src/components/footnote/footnote.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        class="footnote"
+        role="note"
+        aria-labelledby={`footnote-label-${this.identifier}`}
+      >
+        <span id={`footnote-label-${this.identifier}`} class="footnote-id">
+          [{this.identifier}]
+        </span>
+        <div class="text">
+          <span class="footnote-text">{this.text}</span>
+          <a href={`#ref${this.identifier}`} aria-label="Back to main text">
```
Comment (yanis-git) [inline packages/footnote/src/components/footnote/footnote.tsx:26]: our main target is french company, we should not have hard coded english text. It should be at list customizable

File: packages/footnote/src/components/footnote/footnote.spec.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  it('renders', async () => {
+    const page = await newSpecPage({
+      components: [Footnote],
+      html: `<ip-footnote></ip-footnote>`,
+    });
+    expect(page.root).toEqualHtml(`
+            <ip-footnote>
+                <mock:shadow-root>
+                    <div class="footnote" role="note" aria-labelledby="footnote-label-undefined">
+                        <span id="footnote-label-undefined" class="footnote-id">[]</span>
```
Comment (yanis-git) [inline packages/footnote/src/components/footnote/footnote.spec.ts:14]: as you can see here, undefined is used as identifier. seams to miss mandatory parameter checking here or a random string as fallback

File: packages/footnote/src/components/footnote/footnote.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+}
+svg {
+  width: var(--svg-size);
+  height: var(--svg-size);
+  fill: var(--svg-color);
+}
+.text {
+  display: flex;
+  align-items: flex-end;
+}
```
Comment (yanis-git) [inline packages/footnote/src/components/footnote/footnote.scss:40]: you will get css selector collision very easily

**Review by yanis-git — CHANGES_REQUESTED** [review]

File: packages/footnote/src/components/footnote/footnote.tsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  shadow: true,
+})
+export class Footnote {
+  @Prop() identifier = 'default-id';
+  @Prop() btnAriaLabel = 'Back to main text';
+  @Prop() text: string;
+  @Prop() svgColor = 'grey';
+
+  render() {
+    const id = this.identifier || 'fallback-id';
```
Comment (yanis-git) [inline packages/footnote/src/components/footnote/footnote.tsx:15]: this fallback will never being applied as you provide a default value for identifier.

Also i have suggested to put random string over static one.

File: packages/show-more/src/components/show-more/show-more.tsx [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+            <slot name="content"></slot>
+          </div>
+        )}
+        <div class="button-container">
+          <button
+            onClick={() => this.toggle()}
+            style={{ '--svg-color': this.svgColor }}
+            aria-label={this.isExpanded ? 'Show Less' : 'Show More'}
+          >
+            {this.isExpanded ? 'Show Less' : 'Show More'}
```
Comment (yanis-git) [inline packages/show-more/src/components/show-more/show-more.tsx:30]: english text which could not be translated

File: tsconfig.base.json [hunk 8]
```diff
@@ -16,12 +16,18 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
+      "@ipedis/accordion": ["dist/libs/accordion"],
+      "@ipedis/accordion/*": ["dist/libs/accordion/dist/components/*"],
+      "@ipedis/accordion/loader": ["dist/libs/accordion/loader"],
```
Comment (yanis-git) [inline tsconfig.base.json:21]: path

File: tsconfig.base.json [hunk 9]
```diff
@@ -16,12 +16,18 @@
...
       "@ipedis/checkbox": ["dist/packages/checkbox"],
       "@ipedis/checkbox/*": ["dist/packages/checkbox/dist/components/*"],
       "@ipedis/checkbox/loader": ["dist/packages/checkbox/loader"],
       "@ipedis/dropdown": ["dist/packages/dropdown"],
       "@ipedis/dropdown/*": ["dist/packages/dropdown/dist/components/*"],
       "@ipedis/dropdown/loader": ["dist/packages/dropdown/loader"],
+      "@ipedis/footnote": ["dist/packages/footnote"],
+      "@ipedis/footnote/*": ["dist/packages/footnote/dist/components/*"],
+      "@ipedis/footnote/loader": ["dist/packages/footnote/loader"],
```
Comment (yanis-git) [inline tsconfig.base.json:30]: path

File: tsconfig.base.json [hunk 10]
```diff
@@ -31,6 +37,9 @@
       "@ipedis/radio": ["dist/packages/radio"],
       "@ipedis/radio/*": ["dist/packages/radio/dist/components/*"],
       "@ipedis/radio/loader": ["dist/packages/radio/loader"],
+      "@ipedis/show-more": ["dist/libs/show-more"],
+      "@ipedis/show-more/*": ["dist/libs/show-more/dist/components/*"],
+      "@ipedis/show-more/loader": ["dist/libs/show-more/loader"],
```
Comment (yanis-git) [inline tsconfig.base.json:42]: path

File: packages/show-more/src/components/show-more/show-more.scss [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+button:hover {
+  background-color: var(--secondary-color);
+}
+
+svg {
+  width: 20px;
+  height: 20px;
+  fill: var(--svg-color);
+}
```
Comment (yanis-git) [inline packages/show-more/src/components/show-more/show-more.scss:53]: there is risk of css selector collision here

**Review by yanis-git — CHANGES_REQUESTED** [review]

**Pull request merged**